regtests/Test/bigdev.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
changing this to require bash seems unnecessary  - POSIX sh supports `set -e` too.

Reply:
Yes, you are right. I overlooked it somehow, sorry... (thanks for pointing out)

Actually, this is something that can be done in some better way. I added `set -e` because there was no clue that a test is failing. Normally testing frameworks require some result (non-zero) returned for an error.
 
But after the change I made, the remaining tests (in the same script, after failing one) are not executed - what is not good either... Executing them all would require eg. removing `set -e` but instead preserve info about any failure and return proper status at the end. Or some other way (probably making the tests more independent, run them separately).

Well, I guess this just makes points for eventual discussion about improving it...

Anyway, let me know if you want me to revert it and make another merge request or whatever.
(for me, it seems that the easiest/best would be, as you pointed out, just to make the correction bash -> sh (making a new commit), of course, if the other changes will be accepted...).